test/externalTests/zeppelin.sh

```diff
@@ -111,6 +112,9 @@ function zeppelin_test
     # Could result in an error in the future.
     sed -zi "s|it(\('deposit'\)|it.skip(\1|3" test/token/ERC20/extensions/ERC4626.test.js
     sed -zi "s|it(\('withdraw'\)|it.skip(\1|3" test/token/ERC20/extensions/ERC4626.test.js
+    # Avoid the testToInt(248) and testToInt(256) case
```

Comment:
Even with this comment I had to look into the source to understand what's happening. Maybe we can tweak it to be clearer?
```suggestion
    # Here only the testToInt(248) and testToInt(256) cases fail so change the loop range to skip them
```
I think it wasn't clear to me mostly because I did not realize that `testToInt()` runs the test case. Though probably I should have guessed that from the name but I actually expected this to be a bit of code inside one of the test cases rather than something that runs test cases themselves...